### DISCUSSION TITLE
Addittional contributors/help?

### DISCUSSION THREAD

**Discussion by islandmix** [HiEventsDev/Hi.Events#249] [General]
Just curious... how do i check who else is contributing to the project? I saw one contribution recently about a ticket qty item but that's it... 

How do we get more devs on this project? The biggest backend change on roadmap I saw is the "ticket as a product" item which is huge... but Android/Apple wallets, PDF Downloads, AddressField Requirements are core feature items that have not been assigned yet. 

Anhow.. what can we do to get more devs involved? How can the non technical folks help.

**Comment by daveearley** [comment]
Currently, there aren’t many contributors to the project—it’s mostly a one-person effort, which is unfortunately common for open-source projects of this size.

Development is moving slowly at the moment because the "ticket as a product" feature requires a significant code refactor and is taking up most of my time. Once that’s released in a few weeks, I’ll focus on some of the smaller features you mentioned, and updates should come more frequently.

As for getting more developers involved, it’s hard to say. Contributions often happen when a company needs a specific feature and then shares it back. However, more often than not, people fork the code without contributing back.

**Comment by islandmix** [comment]
Thnx for the response... yah i figure that that the "ticket as a product" is taking up most of your time... I will poke some devs to see if there is interest to contribute... but definitely thanks for your efforts!
